docs/installation.md

```diff
@@ -40,3 +40,34 @@ The base installation requires the MuJoCo physics engine (with [mujoco](https://
    ```sh
    $ python robosuite/demos/demo_random_action.py
    ```
+
+### Installing on Windows
+
+It is common to run into issues when installing Robosuite on a Windows machine. Robosuite can be installed on Windows using the following steps.
```

Comment:
should be **robosuite** (in bold) like the rest of the documentation does already